### ISSUE TITLE
Does it support segment a certain type of objects by click one of them?

### ISSUE THREAD

**Issue by li123** [MIC-DKFZ/napari-sam#13]
for example, find all the cat by click one of them?

**Comment by Karol-G** [comment]
Yes, SAM is able after a sufficient number of inputs to deduce what type of object should be segmented and will subsequently do so. However, that does usually not happen after the first click. Usually after you clicked on 2-3 cats then it will understand to segment all cats.
